site/docs/components/checkbox/usage.mdx

```diff
@@ -0,0 +1,45 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Checkbox component
+
+#### When to use Checkbox
+
+- To present an independent choice that the user can select or deselect.
+- To present a list of independent options where the user can select any number of choices.
+
+#### When not to use Checkbox
+
+- When the checkbox displays a mutually exclusive choice between two or more options. Instead, use [Radio Button](../radio-button).
+- To display a single option but trigger a state change directly and immediately. Instead, use [Switch](../switch).
+
+### Import
+
+{/* Update the text and code snippet below as needed: */}
+
+To import Checkbox from the core Salt package, use:
+
+```js
+import { Checkbox, CheckboxGroup } from "@salt-ds/core";
+```
+
+### Props
+
+{/* Update packageName and componentName below as needed */}
+{/* packageName is optional and defaults to "core" if omitted */}
+
+<PropsTable packageName="core" componentName="Checkbox" />
+
```

Comment:
should you add props for CheckboxGroup too?

Reply:
great spot - have added